### PR TITLE
Add verbose flag to SPM

### PR DESCRIPTION
**Pull request by techhat**
Ping @ssgward.

### CODE REVIEW

File: salt/spm/__init__.py [hunk 1]
```diff
@@ -497,24 +498,32 @@ def _remove(self, args):
...
                 # Leave directories in place that still have files in them
-                log.trace('Cannot remove directory {0}, probably not empty'.format(dir_))
+                self._verbose('Cannot remove directory {0}, probably not empty'.format(dir_), log.trace)
 
         self._pkgdb_fun('unregister_pkg', package, self.db_conn)
 
+    def _verbose(self, msg, level=log.debug):
+        '''
+        Display verbose information
+        '''
```
Comment (cachedout) [inline salt/spm/__init__.py]: This is clever. I like it.

**Comment by techhat** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
@techhat Based on the test results, I'm not sure that the flag you've added is being merged into opts correctly. Could you please take another look?

**Comment by cachedout** [comment]
Fixed and merged. Thanks, @techhat

**Pull request merged**